goldeneye.py: add page (#9277)
tldr-pages/tldr: pages/linux/goldeneye.md

> # goldeneye

Reviewer: Page name should match the command name, please rename the file to `goldeneye.py.md` too
```suggestion
# goldeneye.py
```

> # goldeneye

Reviewer: Page name should match the command name, please rename the file to `goldeneye.py.md` too
```suggestion
# goldeneye.py
```

> # goldeneye.py

Author: Please check now

> > A http DoS test tool.

Reviewer 2: ```suggestion
> A HTTP DoS test tool.
```
 We can use this in order to better align with the main page.

> More information: <https://github.com/jseidl/GoldenEye>.

- Test a specific website:

> `./goldeneye.py {{website_url}}"`

Reviewer 2: ```suggestion
`./goldeneye.py {{url}}"`
```

> - Test a specific website with 100 user agents and 200 concurrent sockets:

Reviewer 2: ```suggestion
- Test a specific website with 100 [u]ser agents and 200 concurrent [s]ockets:
```

Reviewer 3: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

> `./goldeneye.py {{url}} -u {{100}} -s {{200}}"`

Reviewer 2: ```suggestion
`./goldeneye.py {{url}} --useragents {{100}} --sockets {{200}}"`
```
We prefer longer arguments in tldr, whenever possible we add mnemonics in command descriptions to highlight the short arguments.

> `./goldeneye.py {{url}} -u {{100}} -s {{200}}"`

Reviewer 3: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --useragents 100 --sockets 200`
```

- Unbalanced trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Redundant curly braces for integer literals 100 and 200.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).
- This fix is optional: no style guide recommendation exists for this case.

> `./goldeneye.py {{website_url}} -u {{100}} -s {{200}}"`

Reviewer 2: ```suggestion
`./goldeneye.py {{url}} -u {{100}} -s {{200}}"`
```

> - Test a specific website without verifying ssl certficate:

Reviewer 2: ```suggestion
- Test a specific website without [n] verifying the SSL certificate:
```

Reviewer 3: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

Reviewer: ```suggestion
- Test a specific website without verifying SSL certificate:
```

> `./goldeneye.py {{url}} -n"`

Reviewer 2: ```suggestion
`./goldeneye.py {{url}} --nosslcheck"`
```

> `./goldeneye.py {{url}} -n"`

Reviewer 3: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --nosslcheck`
```

- Unbalanced trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).

> `./goldeneye.py {{website_url}} -n"`

Reviewer 2: ```suggestion
`./goldeneye.py {{url}} -n"`
```

> - Test a specific website in debug mode:

Reviewer 2: ```suggestion
- Test a specific website in [d]ebug mode:
```

Reviewer 3: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

> `./goldeneye.py {url}} -d"`

Reviewer 2: ```suggestion
`./goldeneye.py {url}} --debug"`
```

> `./goldeneye.py {url}} -d"`

Reviewer 3: The following code fix could be applied:

```suggestion
`./goldeneye.py '{{url}}' --debug`
```

- Unbalanced closing `}` and trailing quote.
- Missed quotes for `{{url}}` placeholder: they required to protect url to be treated as a glob pattern.
- Long options should be used when they exist and portable (portability definition should be written in our style guide).

> - Display help:

Reviewer 2: ```suggestion
- Display [h]elp:
```

Reviewer 3: - Mnemonics are redundant for long options.
- Such case is not handled by our style guide.

> `./goldeneye.py {{website_url}} -d"`

Reviewer 2: ```suggestion
`./goldeneye.py {url}} -d"`
```

> `./goldeneye.py -h`

Reviewer 2: ```suggestion
`./goldeneye.py --help`
```

> `./goldeneye.py -h"`

Reviewer 2: ```suggestion
`./goldeneye.py -h`
```
